src/movies.ts

```diff
@@ -80,8 +83,14 @@ function printMovieTitles(movies: Movie[]): void {
  */
 function countMoviesByYear(movies: Movie[], year: number): number {
   // write your code here...
-
-  return -1; // replace -1 with what you see is fit
+let counter = 0
+movies.forEach(element => {
+  if(element.year === year)
+  {
+    counter++;
+  }
+});
+  return counter; // replace -1 with what you see is fit
```

Comment:
This should have been a `filter` instead of a `forEach`:

```ts
function countMoviesByYear(movies: Movie[], year: number): number {
  return movies.filter((movie) => movie.year === year).length;
}
```